src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -115,11 +115,18 @@ public static double Abs(Complex value)
                 return double.PositiveInfinity;
             }
 
+            return Hypot(value._real, value._imaginary);
+        }
+
+        private static double Hypot (double a, double b)
+        {
             // |value| == sqrt(a^2 + b^2)
             // sqrt(a^2 + b^2) == a/a * sqrt(a^2 + b^2) = a * sqrt(a^2/a^2 + b^2/a^2)
             // Using the above we can factor out the square of the larger component to dodge overflow.
-            double c = Math.Abs(value._real);
-            double d = Math.Abs(value._imaginary);
+            // Really this should be public on Math; it is a very standard numerical method.
```

Comment:
Instead of leaving a comment here, please file an issue here so that we can consider adding this as a public method. It does seem like a fairly reasonable thing to expose, but we'd need to follow the [API review process](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md) before that.